src/database/sqlite/sqlite_health.c

```diff
@@ -301,7 +388,16 @@ bool sql_update_removed_in_health_log_detail(uint32_t unique_id, uint32_t max_un
     return (param == 0 && rc == SQLITE_DONE);
 }
 
-void sql_inject_removed_status(
+#define SQL_INJECT_REMOVED                                                                                                      \
+    "INSERT INTO health_log_detail (health_log_id, unique_id, alarm_id, alarm_event_id, updated_by_id, updates_id, when_key, "  \
+    "duration, non_clear_duration, flags, exec_run_timestamp, delay_up_to_timestamp, info, exec_code, new_status, old_status, " \
+    "delay, new_value, old_value, last_repeat, transition_id, global_id, summary) "                                             \
+    "SELECT health_log_id, @max_unique_id, @alarm_id, @alarm_event_id, 0, @unique_id, UNIXEPOCH(), 0, 0, flags, "               \
+    " exec_run_timestamp, UNIXEPOCH(), info, exec_code, -2, "                                                                   \
+    " new_status, delay, NULL, new_value, 0, @transition_id, NOW_USEC(0), summary FROM health_log_detail "                      \
+    " WHERE unique_id = @unique_id AND transition_id = @last_transition_id RETURNING health_log_id, old_status"
+
```

Comment:
Some assigns in this query called my attention:

| Insert fields | Select Fields |
|---------------|----------------|
|health_log_id | health_log_id| 
| unique_id | @max_unique_id| 
| alarm_id | @alarm_id| 
| alarm_event_id|  @alarm_event_id|
| updated_by_id | 0 |
| updates_id | @unique_id| 
|when_key |UNIXEPOCH() |
| duration| 0 |
|non_clear_duration| 0 |
|flags| flags |
|exec_run_timestamp| exec_run_timestamp| 
|delay_up_to_timestamp| UNIXEPOCH() |
|info|  info |
|exec_code| exec_code |
|new_status| -2 | 
|old_status| new_status|
|delay|  delay |
|new_value| NULL |
|old_value| new_value
|last_repeat| 0 |
|transition_id| @transition_id |
|global_id| NOW_USEC(0)|
|summary|   summary |

Probably this happens due the name table `health_log_detail` and field names. We are assign to `old_*` fields named `new_*`. Also we use the magic number `-2`. 

It does not look like wrong, but probably we could use different names for fields in `health_log_detail`.

Reply:
We can't change health_log_detail easily this will break things rolling back versions